apps/client/src/views/cuesheet/CuesheetPage.tsx

```diff
@@ -53,29 +53,44 @@ export default function CuesheetPage() {
         return;
       }
 
-      const previousValue = event.custom[accessor];
-
-      if (previousValue === payload) {
+      // is it a custom fields
```

Comment:
question: this function is becoming a bit hairy, do you think we could clean improve this?